nicoRDL/dotTheory-JS
# JS-103 (JS-347) JavaScript Functions: Arrow Functions

## Related Ticket

https://codeandtheory.atlassian.net/browse/JS-347

## Description 

- completed Arrow Function tutorial
- completed all tutorial quizzes

## Steps to Validate
1. Open the index file, make sure the random.js is added
2. Open the dev tools console; a random number should appear based on the 2 function calls

## Risks
N/A
